app-crypt/moolticute/moolticute-0.43.19-r1.ebuild

```diff
@@ -1,24 +1,24 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 if [[ ${PV} == 9999* ]]; then
 	EGIT_REPO_URI="https://github.com/mooltipass/moolticute.git"
 	inherit git-r3
 else
-	SRC_URI="https://github.com/mooltipass/moolticute/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://github.com/mooltipass/moolticute/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm"
 fi
 
-inherit xdg-utils qmake-utils udev
+inherit systemd xdg-utils qmake-utils udev
```

Comment:
Please sort the eclasses alphabetically.  Also move this line above if [[ ...

If you look into `skel.ebuild`, inherit should come right after `EAPI`.